### ISSUE TITLE
Go 1.13 Cannot update library repos without main package

### ISSUE THREAD

**Issue by moorara** [renovatebot/renovate#4476]
**What Renovate type are you using?**

We're using the SaaS version of Renovate from GitHub marketplace.

**Describe the bug**

For repositories that they are just library, are imported in other repos, and don't have a `main` package in root level, Renovate will give the following error:

```
Command failed: docker run --rm -v /mnt/renovate/gh/moorara/observe:/mnt/renovate/gh/moorara/observe -v /tmp/renovate-cache/others/go:/tmp/renovate-cache/others/go -e GOPATH -e GOPROXY -e CGO_ENABLED=0 -w /mnt/renovate/gh/moorara/observe renovate/go bash -c "git config --global url.\"https://x-access-token:**redacted**@github.com/\".insteadOf \"https://github.com/\" && go get"
go get .: path /mnt/renovate/gh/moorara/observe is not a package in module rooted at /mnt/renovate/gh/moorara/observe
```

Here is another example:

```
Command failed: docker run --rm -v /mnt/renovate/gh/***/**:/mnt/renovate/gh/***/** -v /tmp/renovate-cache/others/go:/tmp/renovate-cache/others/go -e GOPATH -e GOPROXY -e CGO_ENABLED=0 -w /mnt/renovate/gh/***/** renovate/go bash -c "git config --global url.\"https://x-access-token:**redacted**@github.com/\".insteadOf \"https://github.com/\" && go get"
go get .: path /mnt/renovate/gh/***/** is not a package in module rooted at /mnt/renovate/gh/***/**
```

**Did you see anything helpful in debug logs?**

**To Reproduce**

1.  Create a Go repository with sub-directories and not package at root
2.  Trigger Renovate to update one the go modules

**Expected behavior**

No error and Renovate should update Go repositories with no package at root successfully.

**Screenshots**

**Additional context**

**Comment by rarkins** [comment]
It seems like the `go get` behaviour has changed. If you run `go get` within the root do you observe the same - i.e. it works in Go 1.12 and fails in Go 1.13?

**Comment by moorara** [comment]
@rarkins Yes, with Go `1.13` if I run `go get .` from the root I get the same error.
I think `go get .` only refers to package in current directory and the right command should be `go get ./...` which means current directory and all sub-directories.

**Comment by rarkins** [comment]
FYI the relevant file in source is here: https://github.com/renovatebot/renovate/blob/master/lib/manager/gomod/artifacts.ts

Today we actually just run `go get` without any `.` at all. Do you know if `go get ./...` is backwards compatible with older go versions too?
